ecosystem/sep-0024.md

```diff
@@ -73,6 +74,54 @@ To support the exchange of non-equivalent assets using this protocol, the anchor
 
 Finally, anchors should populate the optional attributes listed in the `GET /transaction(s)` endpoints related to non-equivalent asset transactions, specifically `amount_in_asset`, `amount_fee_asset`, and `amount_out_asset`. This allows client applications can relay this information, in addition to the amounts specified in the associated attributes, back to their users.
 
+Alternatively, the Anchor must support `quote_id` in the `POST /transactions/deposit/interactive` and `POST /transactions/withdraw/interactive`, and the [SEP-38] Anchor RFQ API that enables the Anchor to provide quotes to the Wallet. 
```

Comment:
Given that this is the first time this concept is introduced, I think we can provide some more detail.

```suggestion
Alternatively, the Anchor can support the [SEP-38] RFQ API and the `quote_id` attribute in `POST /transactions/deposit/interactive` and `POST /transactions/withdraw/interactive` requests. This enables wallets to check the Anchor's exchange rates prior to initiating a transaction and obtain a firm quote to use for an upcoming transaction.
```